recipes/starlette-csrf/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "2.0.0" %}
+
+package:
+  name: starlette-csrf
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/s/starlette-csrf/starlette_csrf-{{ version }}.tar.gz
+  sha256: 5f03b8c496840d612f1f7065aac7a209da084972f6e7844bcc5752c32ac54747
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - hatch-regex-commit
+    - hatchling
+    - pip
+    - python >=3.7
+  run:
+    - itsdangerous >=2.0.1,<3.0.0
+    - python >=3.7
+    - starlette >=0.14.2
+
+test:
+  source_files:
+    - tests
+  imports:
+    - starlette_csrf
+  commands:
+    - pip check
+    - mypy -p starlette_csrf
+    - pytest -vv --cov=starlette_csrf --cov-branch --cov-report=term-missing:skip-covered --no-cov-on-fail --cov-fail-under=100
```

Comment:
Coverage tests are probably not needed for a package. This is more a dev tool than a real testing of the package.

Reply:
_probably_: but when i see an upstream that either reports _really high_ coverage, or weird import patterns, i try to put them in :blush: